data/pigui/modules/station-view/06-shipRepairs.lua

```diff
@@ -3,15 +3,153 @@
 
 local ui = import 'pigui/pigui.lua'
 local StationView = import 'pigui/views/station-view'
+local ShipDef = import("ShipDef")
+local Game = import "Game"
+local Rand = import "Rand"
+local InfoFace = import 'ui/PiguiFace'
+local Format = import "Format"
+local Character = import "Character"
+local ModalWindow = import 'pigui/libs/modal-win.lua'
 local Lang = import 'Lang'
-
 local l = Lang.GetResource("ui-core")
 
+local rescaleVector = ui.rescaleUI(Vector2(1, 1), Vector2(1600, 900), true)
+
+local colors = ui.theme.colors
+local icons = ui.theme.icons
+
+local pionillium = ui.fonts.pionillium
+local orbiteer = ui.fonts.orbiteer
+
+local face = nil
+local stationSeed = false
+local damageToRepair = nil
+local repair_cost = 0
+
+local widgetSizes = ui.rescaleUI({
+	itemSpacing = Vector2(4, 9),
+	windowPadding = Vector2(14, 14),
+	faceSize = Vector2(586,565),
+	buttonSize = Vector2(100, 0),
+	verticalDummy = Vector2(0, 50),
```

Comment:
Step 1: Add gauge size to widget sizes:

```suggestion
	verticalDummy = Vector2(0, 50),
	gaugeWidth = 600,
```